src/client/components/Post/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './style.scss';
+import PostHeader from '../PostHeader/';
+import PostInfo from '../PostInfo/';
+import PostAct from '../PostAct/';
+
+class Post extends React.Component {
+  render() {
+    return (
+      <div className='postContainer'>
+        <main className='postBox'>
```

Comment:
main is a much more important element, use [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) or section instead